docs/MAVLINK_FOR_OPENTX.md

```diff
@@ -6,58 +6,62 @@ This page describes how to set up a mLRS system for OpenTx radios running the MA
 
 Four steps need to be completed:
 1. The OpenTx radio needs to be flashed with the MAVLink for OpenTx firmware.
-2. The mLRS Tx module needs to be put into "mBridge mode".
-3. The flight controller needs to be set up for MAVLink on a serial port.
-4. The radio needs to be configured. 
+2. The radio needs to be configured. 
+3. The mLRS Tx module needs to be put into "mBridge mode".
+4. The flight controller needs to be set up for MAVLink on a serial port.
 
```

Comment:
Would a further section with 'Optional but recommended steps' like CRSF page make sense?  I'm not sure if Lua / Yaapu are also supposed to be used on MAVLink for OpenTx....

Reply:
yes, and no. The MAVLink for OpenTx project is bascially a dead horse, since long, and I just updated the page now not because I would consider it important but because I am at it. I.e., the answer is: not worth the effort :)

Native Yaapu does not work here. Risto/rotorman few years ago did a modification to Yaapu so it also would work, but that's a fork of his.

Reply:
Fair enough